test/unified_format.py

```diff
@@ -197,6 +206,18 @@ def __init__(self, observe_events, ignore_commands,
             self._observe_sensitive_commands = False
             self._ignore_commands = _SENSITIVE_COMMANDS | set(ignore_commands)
             self._ignore_commands.add('configurefailpoint')
+        self._event_mapping = {}
+        self.entity_map = entity_map
+        if store_events:
+            for i in store_events:
+                id = i["id"]
+                events = (i.lower() for i in i["events"])
+                for i in events:
+                    if i in self._event_mapping:
+                        self._event_mapping[i].append(id)
+                    else:
+                        self._event_mapping[i] = [id]
```

Comment:
This is the perfect place to use the builtin defaultdict: `self._event_mapping = collections.defaultdict(list)`. Checkout the docs for it to see how we can simplify this code.